docs/usage/configuration.md

```diff
@@ -586,6 +586,30 @@ In most cases you should not need to specify `rename` manually (see the Note bel
     distribution metadata to map module names like 'git' to their distributions
     ('GitPython').
 
+### Dependency Groups (PEP 735)
+
+Tach supports [PEP 735 dependency groups](https://peps.python.org/pep-0735/). By default, Tach will include dependencies from the `dev` dependency group when checking external dependencies.
+
+To configure which dependency groups to include, add `[tool.tach.external]` to your `pyproject.toml`:
+
+```toml
+# In pyproject.toml
+[dependency-groups]
+dev = ["ruff", "mypy"]
+test = ["pytest", "coverage"]
+
+[tool.tach.external]
+include_dependency_groups = ["dev", "test"]
+```
+
+This configuration is read from each package's `pyproject.toml`, making it suitable for monorepos where different packages may have different dependency groups.
```

Comment:
this kind of implies that this is the only part of tach configurable in `pyproject.toml`. i think to be consistent with the rest of the docs it should mention `tach.toml`. there is a separate issue to document that tach can be configured via `pyproject.toml` though (#854)